site/gatsby-site/blog/incident-report-2023-march/index.es.mdx

```diff
@@ -1,69 +1,69 @@
 ---
-title: "AI Incident Roundup – March ‘23"
-metaTitle: "AI Incident Roundup – March ‘23"
-metaDescription: 'Collection of AI incidents, developments, and ingestion in March of 2023, providing a digestible recap on the latest incidents and reports of the AIID.'
+title: "Resumen de incidentes de IA - marzo 2023"
+metaTitle: "Resumen de incidentes de IA - marzo 2023"
+metaDescription: 'Recopilación de incidentes, desarrollos e ingestión de IA en marzo de 2023, que proporciona un resumen digerible de los últimos incidentes e informes de AIID.'
 date: '2023-04-14'
 image: './images/aiid-march.webp'
 author: 'Janet Schwartz & Khoa Lam'
 slug: '/blog/incident-report-2023-march'
```

Comment:
There is a flag that we use to indicate machine-translated content. I think it is `aiTranslated` you can find it on other blog posts, it should be added to the frontmatter.